### DISCUSSION TITLE
Adding Node to Cluster

### DISCUSSION THREAD

**Discussion by ALIEZZ1984** [rabbitmq/rabbitmq-server#11221] [Questions]
During the process of configuring the cluster with IPv6, we've encountered some unexpected hurdles.

We are getting the below error when i try to add my node to a cluster. See error below.

I have confirmed that i have access to all the required ports and can ping the primary node via IP and hostname.

Do i need to do any kind of modifications or configuration since I'm using IPv6 instead of IPv4.

Any suggestions would be appreciated.
Version of RabbitMQ - 3.12.8
Logs from Primary Node1 (ash-genalab304) and Secondary Node2 (ash-genalab305).
[rabbit@ash-genalab304.log](https://github.com/rabbitmq/rabbitmq-server/files/15298254/rabbit%40ash-genalab304.log)
[rabbit@ash-genalab305.log](https://github.com/rabbitmq/rabbitmq-server/files/15298255/rabbit%40ash-genalab305.log)

![330120081-2018c544-b252-436a-9c0e-55aa46a1f93c](https://github.com/rabbitmq/rabbitmq-server/assets/169689982/50ce0340-3559-4ae1-9813-ee80a4046c6d)

**Comment by lukebakken** [comment]
https://www.google.com/search?q=site%3Arabbitmq.com+%22ipv6%22

https://www.rabbitmq.com/docs/networking#distribution-ipv6

According to @mkuratczyk, you're not using the cluster operator, and, since you're on Windows, I'm assuming you're not using kubernetes at all, so I really have no idea why your screenshot shows you running the `kubectl` command.

We will not follow up unless you use the latest version of RabbitMQ, 3.13.2

**Reply by ALIEZZ1984** [reply]
Hi Luke, Yes im not using the Kubectl command you can ignore it from my screenshot, I'm basically just using the rabbitmqctl join_cluster rabbit@ash-genalab304. I looked at the two documents and wasn't sure, about where to adjust those two setting as explained in the networking document, see screenshot
![image](https://github.com/rabbitmq/rabbitmq-server/assets/169689982/b066abc8-7bef-45e6-b55d-197d08ce63ae)
